scripts/build_python_sdk.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+#
+# Builds source and binary Python SDK distributions.
+#
+# Inputs:  ./sdk/python sources generated by the SDK generator; PYPI_VERSION env var set.
+# Outputs: ./sdk/python/bin/dist/{*.tar.gz,*.whl} packages
+
+set -euo pipefail
+
+cd ./sdk/python
+
+# Removing auto-generated setup.py; this is needed until there is an option on the SDK generator to
+# opt out of generating this file.
+rm setup.py
+
+cat << EOF > go.mod
+module fake_python_module // Exclude this directory from Go tools
+
+go 1.17
+EOF
+
+rm -rf ./bin/ ../python.bin/
+cp -R . ../python.bin
+mv ../python.bin ./bin
+
+sed -i.bak -e "s/^  version = .*/  version = \"$PYPI_VERSION\"/g" ./bin/pyproject.toml
+rm ./bin/pyproject.toml.bak
+
+# Modifying generated pyproject.toml until there is an option to populate these settings.
+cat << EOF >> ./bin/pyproject.toml
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+[tool.setuptools.package-data]
+pulumi_kubernetes = ["py.typed", "pulumi-plugin.json"]
+EOF
+
+# Creating an isolated venv for the bulid.
+python3 -m venv venv
+
+source ./venv/bin/activate
+
+python -m pip install pip --upgrade
+python -m pip install wheel build
+
+cd ./bin
+
+# Build the source distribution *.tar.gz.
+python -m build . --sdist
+
+# Build the wheel distribution *.whl.
+python -m pip wheel --use-pep517 --no-deps -w dist .
```

Comment:
1. Is this hand written for this provider? 
2. It looks pretty generic and non-specific to this provider?
3. Could this just be generated by the python codegen instead into `sdk/python/build.sh`?